client/src/routes/modals/ProfileSettingsModal/ProfileSettingsModal.tsx

```diff
@@ -108,7 +108,7 @@ const ProfileSettingsModal = () => {
   );
 
   const remindersPress = useCallback(
-    () => navigate('RemindersModal'),
+    () => navigate('RemindersModal', {}),
```

Comment:
Nit pick... if you in `routes.ts` instead define:
```typescript
RemindersModal: {hideSessionSetting?: boolean} | undefined;
```
You don't have to pass an empty object here.